addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -144,6 +147,8 @@ var FileWidget = SearchableMediaWidget.extend({
 
         this.attachments = [];
         this.selectedAttachments = [];
+        this.libraryMedia = [];
```

Comment:
@sdegueldre 

IS there any good reason to don't use the same type of structure everywhere ? 
Seems like is complexifed due to this list vs set.

Eg: https://github.com/odoo/odoo/pull/56479/files#diff-194923481b66ad4811cdca3dc7b96693R480

It will be probably more clean if we use the same tructure. What do you think about ?

Reply:
Seems your link isn't working. I guess you're wondering why I use a set for selectedMedia and an array for libraryMedia? It's because I need to iterate on libraryMedia to render them, and it's what's returned directly by the API, but for selected media I need to check whether it's already selected or not and toggle it which is much cleaner with a set (rather than splicing an array)

Reply:
selectedAttachments / selectedMedia

here the kind og code that should be simplified using the same structure...

![image](https://user-images.githubusercontent.com/4848797/93209472-78915b80-f75e-11ea-90f3-19c179ff0d3c.png)


Btw, a set is iterable like an list.
Maybe @qsm-odoo has another opinion than me ?

Reply:
Right I see what you're saying now. Considering the number of selected media will never be huge in practice I guess I can just go with an array and keep most of the existing code untouched.